rest/model/project.go

```diff
@@ -551,6 +551,9 @@ type APIGitHubDynamicTokenPermissionGroup struct {
 	Name *string `json:"name"`
 	// Permissions for the GitHub permission group.
 	Permissions map[string]string `json:"permissions"`
+	// AllPermissions is true if the group has all permissions.
+	// This would appear as an empty 'Permissions' map.
```

Comment:
Can you please elaborate more on "// This would appear as an empty 'Permissions' map."? Since empty can also mean no permission, I think more clarification would be helpful.

Reply:
That's a _very_ good question. I'm going to remove this- do you think I should mention that the 'Permissions' map will be ignored?

Reply:
I think this is a good opportunity to explain that the reason we need this boolean is to distinguish between the default empty map and a user specifically requesting an empty map to indicate "no permission".

Reply:
I reworded it to reflect the restrictions on the fields